Makefile

```diff
@@ -0,0 +1,23 @@
+REGISTRY?=registry.devshift.net
+REPOSITORY?=fabric8-analytics/f8a-gemini-server
```

Comment:
The repo name should be changed as per this - fabric8-analytics/f8a-gemini-server